bsconfig.json

```diff
@@ -1,16 +1,24 @@
 {
   "name": "renature",
   "version": "0.1.0",
-  "sources": {
-    "dir": "src/core",
-    "subdirs": true
-  },
+  "sources": [
+    {
+      "dir": "src/core",
+      "subdirs": true
+    },
+    {
+      "dir": "__tests__",
+      "subdirs": true,
+      "type": "dev"
+    }
+  ],
   "package-specs": {
-    "module": "es6",
+    "module": "es6-global",
```

Comment:
https://bucklescript.github.io/docs/en/build-configuration#package-specs

It appears that `es6-global` will allow for relative imports of `node_modules`, which apparently (?) is better for being able to use ESM in the browser. I'm a bit foggy on this part of the BuckleScript docs (glad Patrick is working on them), so @kitten if you understand this better I'd love any insight you may have.

Reply:
Afaik it’s only used in development for browsers since it results in import statements from `./node_modules/pkg/...` explicitly, so not so great for bundling in all cases.

So if you’re planning to have a build system past Bucklescript, I’d stick to the usual 🙂

Reply:
🆒 Sounds good, reverting back to `es6` makes sense to me too based on the BS output.